File: solutions/go/raindrops/1/raindrops.go
package raindrops

import "fmt"

func Convert(number int) string {
    sounds := map[int]string{
        3: "Pling",
        5: "Plang",
        7: "Plong",
    }
	raindrops := ""
	for n, sound := range sounds {
        if isDivisible(number, n){
            raindrops += sound
        }
    }
	if raindrops == "" {
        return fmt.Sprintf("%d", number)
    }
	return raindrops
}

func isDivisible(a, b int) bool {
	return a % b == 0    
}

File: solutions/python/raindrops/2/raindrops.py
def convert(number):
    sounds = {
        3: "Pling",
        5: "Plang",
        7: "Plong",
    }
    raindrop_sound = ''.join([ sound for n, sound in sounds.items() if divisible(number, n)])
    return raindrop_sound or str(number)

def divisible(a: int, b: int) -> bool:
    return True if not a % b else False
